docker/bootstrap/build.sh

```diff
@@ -47,11 +47,9 @@ fi
 chmod -R o=rx *;
 
 arch=$(uname -m)
-[ "$arch" == "aarch64" ] && [ $flavor != "common" ] && arch_ext='-arm64v8'
-
```

Comment:
The above file was the only one using this, so cleaning this up. Any potential future `arm64` support would be done differently anyway.